### PR TITLE
[Compatibility] Added NOVALUE option for HSCAN

### PR DESCRIPTION
**Pull request by Vijay-Nirmal**
- [x] Add NOVALUE option for HSCAN
- [x] Updated Test cases
- [x] Updated documentation
- [x] Updated StackExchange.Redis to `2.8.16` to get the new `HashScanNoValues` method in the test project

### CODE REVIEW

File: libs/server/Objects/Hash/HashObject.cs [hunk 1]
```diff
@@ -228,7 +228,10 @@ public override unsafe void Scan(long start, out List<byte[]> items, out long cu
                 if (patternLength == 0)
                 {
                     items.Add(item.Key);
-                    items.Add(item.Value);
+                    if (!isNoValue)
```
Comment (yrajas) [inline libs/server/Objects/Hash/HashObject.cs:231]: The count value will have to be adjusted as well if NOVALUES present as it is currently always multiplying it by two to account for key and value.

**Review by yrajas — CHANGES_REQUESTED** [review]
Count to be adjusted for hscan.

File: libs/server/Objects/Hash/HashObject.cs (hunk 1 above)
Comment (Vijay-Nirmal) [reply libs/server/Objects/Hash/HashObject.cs:231]: Fixed

**Comment by Vijay-Nirmal** [comment]
@yrajas Fixed the review command and added one more case to test this case

**Comment by Vijay-Nirmal** [comment]
@yrajas Fixed failed test case

**Pull request merged**